packages/block-editor/src/store/reducer.js

```diff
@@ -1778,6 +1778,21 @@ export function lastBlockInserted( state = {}, action ) {
 	return state;
 }
 
+/**
+ * Reducer returning the block that is eding temporarily edited as blocks.
+ *
+ * @param {Object} state  Current state.
+ * @param {Object} action Dispatched action.
+ *
+ * @return {Object} Updated state.
+ */
+export function temporarilyEditingAsBlocks( state = '', action ) {
+	if ( action.type === 'SET_TEMPORARILY_EDITING_AS_BLOCKS' ) {
+		return action.temporarilyEditingAsBlocks;
```

Comment:
I think we should improve the nomenclature here. It doesn't convey much information to say that we are editing [a block] _as blocks_. ;) 

This deserves a bit more thought and I don't have real recommendations yet. One thing that crossed my mind was the more explicit `temporarilyUnlockInnerBlocks`.